docs/doxygen-user/multi-user/installSolr.dox

```diff
@@ -0,0 +1,141 @@
+     - Remove the _CONSOLE_ appender from the _log4j\.rootLogger_ line.
+ <br><br>
+ The log file should end up looking like this (modified lines are highlighted in yellow
+ <br><br>
+ \image html log4j.PNG
+ <br><br>
+5. <b>Schema Configuration</b>: From an Autopsy installation, copy the following into <i>"C:\Bitnami\solr-4.10.3-0\apache-solr\solr"</i>:
+- The folder <i>"C:\Program Files\Autopsy-XXX(current version)\autopsy\solr\solr\configsets"</i>
+- The folder <i>"C:\Program Files\Autopsy-XXX(current version)\autopsy\solr\solr\lib"</i>  
+- The file <i>"C:\Program Files\Autopsy-XXX(current version)\autopsy\solr\solr\zoo.cfg"</i>
+
+
+\subsection install_solr_reinstall Reinstall Service
+
+Because we made changes to the service configuration, we need to reinstall it. 
+
+1. Start a Windows command prompt as administrator by pressing Start, typing <i>command</i>, right clicking on <i>Command Prompt</i>, and clicking on <i>Run as administrator</i>. Then run the following command to uninstall the solrJetty service:
+
+              cmd /c C:\Bitnami\solr-4.10.3-0\apache-solr\scripts\serviceinstall.bat UNINSTALL
+
+       You will very likely see a result that says "The solrJetty service is not started." This is okay.
+2. In the same prompt, run the following command to install the solrJetty service:
+
+               cmd /c C:\Bitnami\solr-4.10.3-0\apache-solr\scripts\serviceinstall.bat INSTALL
+<br> Note the argument "INSTALL" is case sensitive. Your command prompt should look like the screenshot below. Very likely your command prompt will say "The solrJetty service could not be started." This is okay.
+ <br><br>
+ \image html solrinstall1.PNG
+ <br><br>
+ 
+
+You should be able to see the Solr service in a web browser via the URL http://localhost:8983/solr/#/ as shown in the screenshot below.  If you can, you should skip the next step. If you cannot, proceed to the next step.
+
+
+\subsection install_solr_service_user Configure Service User
+
+Back in \ref install_multiuseruser_page, you should have decided what user to run Solr as.   To configure Solr to run as that user, you'll use Windows Service Manager. 
+
+
+- Press _Start_, type _services.msc_, and press _Enter_. 
+- Find _solrJetty_. If the service is running, press _Stop the service_, 
+- Double click the service and switch to the _Log On_ tab to change the logon credentials to the chosen user who will have access to the shared storage. 
+ - If you specify a domain account, the account name will be in the form of _DOMAINNAME\\username_ as shown in the example below
+
+\image html solrinstall2.PNG
+
+
+- Start the service again. 
+
+\section install_solr_test Testing
+
+There are two tests that you should do before declaring that Solr has been configured. 
+
+- <b>Web Interface</b>: You should be able to see the Solr admin panel in a web browser via the URL <i>http://localhost:8983/solr/#/</i>.
```

Comment:
The URL we suggest using here (localhost) will not reveal firewall issues. We should suggest that they attempt to connect from another machine on the network.